### DISCUSSION TITLE
[Feature] <Picture "collection" based on current GPS location>

### DISCUSSION THREAD

**Discussion by soana** [immich-app/immich#9796] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Can we have a button on the left hand menu that can generate a page showing all the pictures from your photo collection that are taken in a certain radius from the current GPS location.
This would be something like a Memories feature but on demand based on GPS location.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4519

**Reply by soana** [reply]
Dang, this feature actually exists already. All I needed to do on the mobile app is to open the map and click on the circle icon in the bottom right corner to zoom in my location. 
Sorry for wasting your time here :(
